packages/data-context/src/actions/WizardActions.ts

```diff
@@ -272,10 +278,12 @@ export class WizardActions {
     // only do this if config file doesn't exist
     this.ctx.lifecycleManager.setConfigFilePath(`cypress.config.${this.ctx.coreData.wizard.chosenLanguage}`)
 
+    const adjective = (testingType === 'e2e') ? 'E2E' : 'component testing'
+
     return this.scaffoldFile(
       this.ctx.lifecycleManager.configFilePath,
       configCode,
-      'Created a new config file',
+      `The Cypress config file where the ${adjective} dev server is configured.`,
```

Comment:
Not sure this wording makes sense for e2e. You _can_ configure a server for e2e inside of `setupNodeEvents` but it's not required. It should just say "The Cypress configuration file for {adjective} testing".